### ISSUE TITLE
Design and implement the new Notary website

### ISSUE THREAD

**Issue by FeynmanZhou** [notaryproject/notaryproject.dev#194]
This issue relates to a proposal for a [LFX mentorship project](https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2023/02-Jun-Aug) for Term 2, 2023.  Please feel free to comment on this issue about your interest and willingness, but note that commenting here is not the same as applying to LFX.

- Description: Design the new Notary website using the Figma tool and develop it based on the design layout. We redesigned the Notary website and finished the first phase development work with CNCF employee @thisisobate in https://github.com/notaryproject/notaryproject.dev/pull/139. This project is to continue to design and implement the new Notary website with a developer-friendly layout design.
- Expected Outcome: 
   - Design and implement the Adopters page
   - Redesign a Community page
   - Improve the landing page design; add an installation section with a terminal effect design
   - Support mobile responsive design
   - Add a pop-up window on the landing page to tell users how to join the community
   - Add Algolia search for the website
   - Design and implement a video page to list Youtube videos
   - Refine the content on the website
   - Add Broken link check to Netlify CI
- Recommended Skills: Figma design, HTML, CSS, JavaScript, Hugo
- Mentor(s):  Feynman Zhou (@FeynmanZhou , feynmanzhou@microsoft.com)

**Comment by MohdSultanGit** [comment]
Hello @FeynmanZhou I am Mohd Sultan I am very excited to participate in this project under lfx mentorship  term-2. 
Currently i am learning and working on HTML, CSS, Javascript, FIGMA, etc. So, I would love to contribute in this project under your mentorship.
Can you can give me any resources, slack channel or any other required materials to work on this project ,
Looking forward to work on this project 
Thanks,
Mohd Sultan

**Comment by money8203** [comment]
Hey @FeynmanZhou I'd love to work on this project under LFX mentorship term 2. I've had experience building websites in HTML, CSS and JavaScript and figma to create a developer-friendly layout design. Not familiar with Hugo tho, can learn along the way.

Could you provide me with some resources to get started with initially so that I ca get myself familiarized with the project!

**Comment by 1Shubham7** [comment]
Hello @FeynmanZhou, This is Shubham.
First of all, I'd like to express my sincere gratitude for your mentorship in the LFX program. I am a first-year undergraduate student studying Computer Science Engineering in India. I have experience in web development, Android development, and some knowledge in DevOps. I am skilled in both front-end (HTML, CSS, JavaScript, React) and backend ( Django ), and other technologies that are not directly related to this project. I am also learning Figma, and I am very excited to apply for LFX term 2 under this project. Could you please provide me with some resources that will help me get familiar with the code base? Also, I would appreciate it if you could share the link to the Slack channel.
Thank you so much for your time and support!

**Comment by PranitPatil03** [comment]
Hello @FeynmanZhou  👋

I'm Pranit Patil. I am pursuing a Bachelor's degree in Computer Science from Mumbai University and working as a frontend development intern at Hubx.ai. I have previously worked as a front-end developer intern at Tech Cryptors. 

I am proficient in JavaScript, ReactJS, Node, Express, MongoDB, and CSS libraries and frameworks like Bootstrap. I have worked on multiple projects, including full-stack projects, which have helped me develop a strong foundation in Web development. 

I'm interested in joining the LFX mentorship program and becoming an active member of the community. Could you please assist me in getting involved in project discussions? Also, could you share the Figma Design and some resources for understanding Hugo? Lastly, I'm having difficulty joining the Slack group.

Thank you

**Comment by ankush9302** [comment]
i'm ankush kurmi pursuing  bachelor of technology.i am a front end developer.

i am proficient in javascript , css, html. currently i am  learning some backend stuff like Node,Express,mongodb.

i want to contribute to this project . can you please take me under your guidance and give me a chance to work .

**Comment by ShravaniAK** [comment]
Hello ,I am Shravani Kaware , i am pursuing my bachelor of technology from IIIT Vadodara , I am a frontend developer and i am proficient in Html , css , Javascript , React js, Next js and frameworks like Hugo .
I am very much interested in contributing to this project .
Thankyou

**Comment by BabyElias** [comment]
Hello @FeynmanZhou !
This is Anoushka Jha , a sophomore and tech-enthusiast from IIT(BHU), Varanasi, India. I have experience in designing using Figma and Canva and have made numerous open-source contributions using HTML, CSS, JavaScript and frontend frameworks React & Vue. Js. I am really excited to apply for this project under LFX Mentorship Term-2.

Forked and setup the project locally already (faced some issues with the nvm install --lts command, resolved later)
Gone through the codebase and file architecture in order to get an overview of the website.


Have some ideas for improving on the site and would love to discuss them through any suitable communication channel !
Had some issues joining Slack channel using the link provided on website.

**Comment by SiddheshKukade** [comment]
Hello👋 , SiddheshKukade here I'm a professional developer and engg student from PVGCOET India.I've been participated twice in Google Summer of Code program and I have almost 10 months of work experience.

I am willing to apply for LFX mentorship program. 

here's my GitHub and LinkedIn: 
https://github.com/SiddheshKukade
https://in.linkedin.com/in/siddhu-kukade
Feel free to connect 🤠

**Comment by Ajiboso-Adeola** [comment]
Hello @FeynmanZhou I am Adeola Ajiboso. I am very excited to participate in this project under lfx mentorship term-2.
Currently i am learning and working on HTML, CSS, Javascript, FIGMA, etc. So, I would love to contribute in this project under your mentorship.
Can you can give me any resources, or any other required materials to work on this project ,
Looking forward to work on this project
Bestregards,
Adeola Ajiboso

**Comment by symys** [comment]
Hello @FeynmanZhou and everyone, I'm Seyma👋  

I am a frontend developer with urban planning background who is detail-oriented, curious and willing to learn new technologies every day. I just graduated from a 5 month Re:Coded frontend web development bootcamp, which is an US-based platform with participants from different countries, and I gained a lot of experience in terms of working as a team as well as technical skills. 

I have knowledge about HTML, CSS, JavaScript, React, Bootstrap, Tailwind, Redux Toolkit and responsive design and I continue to improve myself via developing projects. I had some experience with Figma and I think your project is a great opportunity to develop my Figma skills. Besides, I'm excited to add HUGO to my talents. I would love to work on this project under LFX mentorship term-2 with your mentorship.

Looking forward to work on this project

Thanks,
Seyma Demir

**Comment by AhmedBedeir** [comment]
Hello @FeynmanZhou👋

I hope this message finds you well. I am Ahmed Bedeir I have a strong enthusiasm for web design and development, and I believe this project would be an excellent opportunity for me to enhance my skills in Figma design, HTML, CSS, JavaScript, and Hugo. I am eager to contribute to the redesign of the Adopters and Community pages, improve the landing page design with a terminal effect, implement a pop-up window for user guidance, and design and implement the video page.

As a frontend developer, I know the importance of agile methodologies and adhering to task delivery timelines. I am comfortable working in an agile environment, where iterative development and frequent collaboration are the norms. I am committed to delivering work and meeting project deadlines, as I understand the significance of timely and efficient delivery in achieving project success.

Thank you for considering my interest in this project. I would greatly appreciate the opportunity to participate in the LFX mentorship program and contribute to the Notary website redesign. Please feel free to reach out to me if you have any further questions or need additional information.

Thanks,
Ahmed Bedeir

**Comment by Prakhar-Shankar** [comment]
Hello @FeynmanZhou 

I am Prakhar Shankar, I am a Passionate Web Developer adept in bringing forth expertise in creating, designing and maintaining
websites. Equipped with a diverse and promising skill-set. Proficient in an assortment of technologies
including HTML, CSS, Tailwind CSS, Vanilla JavaScript, ReactJS, NodeJS, ExpressJS, MongoDB,PHP,
SQL,and C/C++. Able to effectively self-manage during independent projects, as well as collaborate in a
team setting.

It will be an honour to work with such cool mentors, I have also tried solving some issues. I have started reading Hugo's documentation.  Thanks for going through this message.

Looking forward to work on this project.

Regards,
Prakhar

**Comment by Srishti-j18** [comment]
Hello @FeynmanZhou ,

Myself Srishti Jaiswal, a second-year student from the Indian Institute of Technology (BHU) Varanasi.Also I have been selected for a GSoC'23 project through Summer of Reproducibility with UC OSPO (University of California). I am very interested in participating in the LFX mentorship program with "CNCF - Notary: Design and implement the new Notary website". I have submitted my task on the LFX site. And I am interested to learn and enhance my skill under your amazing guidance. Please tell me what I can do next, as I submitted my resume and cover letter on the website.

Thank you!

**Comment by arunkarthicknm** [comment]
Hello @FeynmanZhou !!
Myself Arun karthick, a front end developer specialized in React.js from India. I've previously contributed to CNCF projects like Meshery and Glossary. I would like to contribute to the notary website  project. Kindly consider my proposal. Looking forward to hear from you.
Regards,
Arun karthick N M.

**Comment by toukirkhan** [comment]
Applied for the project! Looking forward to make some impactful contribution.

Regards,
Toukir

**Comment by kirtanchandak** [comment]
Hey @FeynmanZhou, really excited to work on this project, and create an amazing community website that users would love. 
I am good at HTML, CSS, JavaScript, ReactJS, Bootstrap, Tailwind CSS, ExpressJS and MongoDB. I have gone through the previous PR made by @thisisobate and understood the complete flow of how things are falling into place and I am excited to build further features on top of it.  

See you on the other side!!

Regards, 
Kirtan Chandak.

**Comment by kirtanchandak** [comment]
Hey @FeynmanZhou, 

Really excited to work on this project, and create an amazing community website that users would love.
I am good at HTML, CSS, JavaScript, ReactJS, Bootstrap, Tailwind CSS, ExpressJS and MongoDB. I have gone through the previous PR made by @thisisobate and understood the complete flow of how things are falling into place and I am excited to build further features on top of it.

See you on the other side!!

Regards,
Kirtan Chandak.

**Comment by Pranjal7852** [comment]
Hello! I'm Pranjal Goyal from the National Institute of Technology, Hamirpur, and I'm a pre-final year undergraduate student studying electronics and communication engineering. I'm from New Delhi, and I love to learn about new technology and work on challenges that affect the actual world. I have worked on UI/UX design in the past, but my current area of specialisation is full stack web development in MERN stack.  I am good at making stuff that people say I can't. The thing that drives me the most is my curiosity and my eagerness to learn as much as I can.

I want to work with notary on this project for the summer, as I think I have the required skills to bring this project to its completion, and I am comfortable in learning things on the fly.  I have already started discussing the objectives of the project with the community and started learning the preliminaries for the project. I can assure you that if I get selected to work with Notary this summer, I will definitely try my level best to make this project successful.

Looking forward to working with you :)

**Comment by DIWAKARKASHYAP** [comment]
i applied in LFX , what i do next

**Comment by OlaoyeVic** [comment]
Dear @FeynmanZhou ,

I hope this message finds you well and filled with enthusiasm for the exciting project ahead. As a frontend developer with a burning passion for web development, I am thrilled to throw my hat in the ring for the "Design and implementation of the new Notary website" opportunity. Allow me to showcase how my skills, enthusiasm, and past accomplishments make me an ideal candidate to contribute to this transformative project.

With my proficiency in HTML, CSS, JavaScript, React, and TypeScript, I have embarked on various noteworthy endeavors that demonstrate my ability to craft compelling web applications. One of my standout projects was a project I collaborated with two other developers to build. It was an Ed-tech SaaS platform, designed to empower users to learn and earn simultaneously. Leveraging the power of React, I developed a feature-rich platform complete with a library, a dashboard, and robust login and signup capabilities. This project not only highlighted my technical prowess but also showcased my commitment to creating engaging, user-friendly experiences that cater to the needs of modern learners.

Furthermore, I have also taken the reins in building the entire frontend of a transaction dashboard. This experience allowed me to deepen my understanding of data visualization, user interaction, and performance optimization, equipping me with valuable skills that I am eager to apply to the Notary project. I thrive on the challenges presented by complex UI implementations and find immense satisfaction in transforming ideas into polished, functional interfaces.

Here's why I believe my inclusion in your team will yield exceptional results:
1. Enthusiasm and Eagerness to Learn
2. Team Spirit
3. Ability to adapt quickly and be resourceful
4. Strong foundation


I am genuinely excited about the prospect of joining your team for the LFX Mentorship Program and contributing my skills to revolutionize the Notary website.  Thank you for considering my application, and I look forward to the chance to contribute my talents to the success of the Notary project.

Best regards,
Victor Olaoye.
